olsrd/Makefile

```diff
@@ -193,6 +193,8 @@ define Package/olsrd/install
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_BIN) ./files/olsrd4.init $(1)/etc/init.d/olsrd
 	$(INSTALL_BIN) ./files/olsrd6.init $(1)/etc/init.d/olsrd6
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DATA) ./files/olsrd-neigh.sh $(1)/usr/bin/olsrd-neigh.sh
```

Comment:
I would suggest you make a seperate package and call it olsrd-utils-neighbors or something like this.

Reply:
I don't agree. A separate package for just one script seems looks like overkill to me. Maybe we should postpone that new package until there are additional scripts.